torch/onnx/utils.py

```diff
@@ -747,6 +742,14 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
                         opened_file.write(v)
             else:
                 raise RuntimeError("Unknown export type")
+
+            # The ONNX checker only works for ONNX graph. So if the operator_export_type is not ONNX,
+            # we can skip this check.
+            # If large model format export is enabled, proto will only contain data location instead of
+            # raw data and _check_onnx_proto() will fail because it can only handle the raw ONNX proto
+            # string in memory.
+            if (operator_export_type is OperatorExportTypes.ONNX) and (not val_use_external_data_format):
```

Comment:
Let's raise a more specific type of exception to make it easier for users to catch specific exceptions if they want to reproduce the old behavior of disabling the checker.

Something like:

```py
class ONNXCheckerError(Exception):
    pass


try:
   _check_onnx_proto(proto)
except RuntimeError as e:
  raise ONNXCheckerError(e)
```

And add this to the doc string of export() in a new "Raises" section after the "Args" section according to this style:
https://google.github.io/styleguide/pyguide.html#doc-function-raises